Typeracer_Game/src/typeracer_game/Game.java

```diff
@@ -70,20 +80,78 @@ private void showPara() {
             e.printStackTrace();
         }
     }
+    
+    public void showScore() {
+        Thread display = new Thread(new Runnable(){
+            @Override
+            public void run() {
+                try {
+                    while (!endThread) {
+                        float accuracy=100;
+                        if (space!=0)
+                            accuracy = (float) ((float) (space - error_count) / (float) space) * 100;
+                        txtAcc.setText(String.format("Accuracy : %.1f ",accuracy));
+                        float spd=current_speed*60;
+                        txtSpeed.setText(String.format("Speed : %.1f wpm",spd));
+                        paraTextPane.select(0,space);
+                        txtWord.setText(correct_word);
+                        if (start_time!=-1&&!paused) {
+                            int time=(int) System.currentTimeMillis()-start_time-pause_time;
+                            time=time/100;
+                            txtTime.setText("Time : "+(time/10)+"."+(time%10)+" sec");
+                        }
+                        if (change) {
+                            highlight(paraTextPane,curPos);
+                            change=false;
+                        }
+                        Thread.sleep(50);
+                    }
+                }
+                catch (Exception e) {
+                    
+                }
+            }
+        });
+        display.start();
+    }
+
+    class MyHighlightPainter extends DefaultHighlighter.DefaultHighlightPainter {
+        public MyHighlightPainter(Color col) {
+            super(col);
+        }
+    }
+    Highlighter.HighlightPainter myHighlightPainter = new MyHighlightPainter(Color.CYAN);
+    public void highlight(JTextComponent textComp,int end) {
+        try {
+            Highlighter hilite = textComp.getHighlighter();
+            Document doc = textComp.getDocument();
+            String text=doc.getText(0,doc.getLength());
+            hilite.addHighlight(0,end,myHighlightPainter);
+        }
+        catch (Exception e) {
 
+        }
+    }
+    
+    
     /**
      * This method is called from within the constructor to initialize the form.
      * WARNING: Do NOT modify this code. The content of this method is always
      * regenerated by the Form Editor.
      */
     @SuppressWarnings("unchecked")
-    // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
```

Comment:
Don't modify these comments so that it remains compatible with Netbeans. Take care of all such lines where these comments were present.